packaging/windows/readme.md

```diff
@@ -137,8 +137,7 @@ Notable executables
 `kbfsdokan.exe` - kbfs
 `kbnm.exe` - browser extension
 `keybase.exe` - service
-`keybase.rq.exe` - quiet launcher and de-elevator
+`keybaserq.exe` - quiet launcher and de-elevator
```

Comment:
Where have we changed the name of the executable?

Reply:
i think this was just a typo. it's `keybaserq.exe` on the file system and everywhere else.